<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#131239</string>
  </property>
  <widget class="QLineEdit" name="f_username">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:0px 10px 0px 10px;
border-radius:5px;
border:3px solid lightgreen;
background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Your Username</string>
   </property>
  </widget>
  <widget class="QLabel" name="f_uname_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Georgia&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f_sendmail">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
padding:15px;
background-color:green;
color:white;</string>
   </property>
   <property name="text">
    <string>Send Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="f_otp_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>550</y>
     <width>401</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Georgia&quot;;
color: white;</string>
   </property>
   <property name="text">
    <string>Enter OTP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_otp">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>600</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:0px 10px 0px 10px;
border-radius:5px;
border:3px solid lightgreen;
background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Your OTP</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f_submit">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>700</y>
     <width>200</width>
     <height>60</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
padding:15px;
background-color:green;
color:white;</string>
   </property>
   <property name="text">
    <string>Submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="f_header_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>40</y>
     <width>400</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 30pt &quot;Copperplate Gothic Light&quot;;
color:white;
</string>
   </property>
   <property name="text">
    <string>OTP verification</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="f_sentmail_label">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>470</y>
     <width>400</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
color:lightgreen;
font: 75 12pt &quot;Courier&quot;;</string>
   </property>
   <property name="text">
    <string>We've sent a verification code to your registered email </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="f_response_otp">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>650</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
color:red;
font: 75 12pt &quot;Courier&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
